### PR TITLE
Molgenis 4.1

### PR DESCRIPTION
**Pull request by djvanenckevort**
Updated Docker files for the latest versions.

### CODE REVIEW

File: molgenis/4.1/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+version: "2.1"
```
Comment (marikaris) [inline molgenis/4.1/docker-compose.yml]: Shouldn't this be 4.1?

**Comment by djvanenckevort** [comment]
> Op 13 jun. 2017, om 11:08 heeft marikaris <notifications@github.com> het volgende geschreven:
> 
> @marikaris commented on this pull request.
> 
> I think you forgot to replace one version tag
> 
> In molgenis/4.1/docker-compose.yml:
> 
> > @@ -0,0 +1,26 @@
> +version: "2.1"
> 
> Shouldn't this be 4.1?

This is the docker-compose.yml version, which is correctly set to 2.1. 
See: https://docs.docker.com/compose/compose-file/compose-versioning/#compatibility-matrix

With kind regards,
-- 
David van Enckevort, MA
Project Manager
Dept of Genetics, UMCG
+31 (0)6 54 33 22 76
Skype: enckevort76

**Comment by marikaris** [comment]
But why is the version in the 3.0 molgenis release then 3.0?

**Comment by djvanenckevort** [comment]
> Op 13 jun. 2017, om 11:15 heeft marikaris <notifications@github.com> het volgende geschreven:
> 
> But why is the version in the 3.0 molgenis release then 3.0?

You are right. I checked with the wrong version (docker-compose instead of docker).

With kind regards,
-- 
David van Enckevort, MA
Project Manager
Dept of Genetics, UMCG
+31 (0)6 54 33 22 76
Skype: enckevort76

**Pull request merged**